treeherder/seta/migrations/0001_squashed_0002_remove_task_request_and_jp_timeout.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
this file, why are we adding it?  is it needed for a sequence of migrations to work properly?

Reply:
It's needed for the second migration to work (Django keeps track of migrations).